TypeCobol.LanguageServer/TypeCobolCustomLanguageServer/TypeCobolCustomLanguageServer.cs

```diff
@@ -59,6 +59,16 @@ protected override void OnDidChangeConfiguration(string[] options)
                 this.UseCfgDfaDataRefresh = UseCfgMode.AsContent;
             else
                 this.UseCfgDfaDataRefresh = UseCfgMode.No;
+#if EUROINFO_RULES
+            if (!options.Contains("-ycpl") && this.CpyCopyNamesMapFilePath != null)
+            {
+                string[] newOptions = new string[options.Length + 2];
+                Array.Copy(options, newOptions, options.Length);
+                newOptions[options.Length] = "-ycpl";
+                newOptions[options.Length + 1] = this.CpyCopyNamesMapFilePath;
+                options = newOptions;
+            }
+#endif
```

Comment:
?? I'm confused. Here we have received updated options from the user in `options` string array, we should not modify its content based on previously stored values.

Reply:
It is not in fact the previous value, but the value that was passed when the server was started, it cane be the default file, but at this moment I don't know if we will manage this option in the client.

Reply:
Either we allow the client to change the path on-the-fly or we only accept value at startup time. If the path may be changed we'll have to check if it's a new value (to avoid loading multiple times the same file).

Reply:
So we decided that, -ycpl should not be a dynamic LSP client configuration option but only a command line LSP server option.